### ISSUE TITLE
About test-domain validation set Score...

### ISSUE THREAD

**Issue by Genie-Kim** [facebookresearch/DomainBed#35]
Thanks for making a great library.
However, while I was measuring the performance of the algorithm with sweep.py, I found that the test-domain validation set came out lower than the training-domain validation set.
It's not just my algorithm, it's common in the existing sweep data.
I think this is not normal, Any idea why this is happening?

**Comment by lopezpaz** [comment]
The "test-domain validation" metric is limited to one query per hyper-parameter (in fact, the last epoch) in order to avoid extremely optimistic results.
